tests/integration/integ_test_base.py

```diff
@@ -150,7 +150,7 @@ def _get_config_file_name(self) -> str:
         config_file = open(os.path.join(self.tmp_dir, 'test.conf'), 'w+')
         config_file.write(
             '[TabPy]\n'
-            f'TABPY_QUERY_OBJECT_PATH = {self.tmp_dir}/query_objects\n'
+            f'TABPY_QUERY_OBJECT_PATH = ./query_objects\n'

```

Comment:
If we create this directory in the root won't it not get cleaned up after the test run?

Reply:
the folder is created in the temp folder which is current folder when the test runs

Reply:
I see, so we don't actually need to specify {self.tmp_dir} for TABPY_STATE_PATH either?

Reply:
Probably not. We change the folder either when starting TabPy with startup script or when pointing to config path.